optics-core/src/Optics/IxFold.hs

```diff
@@ -192,6 +196,8 @@ ifiltered
 ifiltered p o = mkIxFold $ \f ->
   itraverseOf_ o (\i a -> if p i a then f i a else pure ())
 {-# INLINE ifiltered #-}
+-- Note: technically one could filter lens into atraversal, atraversal into
+-- itself filtering atraversal might be useful in fact.
```

Comment:
I don't understand this note, can you clarify?

Reply:
that's the same idea that `filtered` is an affine fold.

Reply:
Ok, but it looks to me like there is some punctuation missing there, I can't parse that sentence :c